app/controllers/application_controller.rb

```diff
@@ -1,2 +1,9 @@
 class ApplicationController < ActionController::Base
+  def current_ability
+    if current_artist
+      @current_ability ||= Ability.new(current_artist)
```

Comment:
Overrides cancancan's default of checking for `current_user`, which doesn't exist in our model